WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -90,9 +89,9 @@ public HistoryViewModel(WalletViewModel walletViewModel, IObservable<Unit> updat
 				x => x.IsExpanded),
 
 				// Date
-				new PrivacyTextColumn<HistoryItemViewModelBase>(
+				new TemplateColumn<HistoryItemViewModelBase>(
 					"Date / Time",
-					x => x.DateString,
+					new FuncDataTemplate<HistoryItemViewModelBase>((node, ns) => new PrivacyContentControl { Content = node.DateString, NumberOfPrivacyChars = 15 }, true),
```

Comment:
The PrivacyTextColumn was specifically used to improve history table performance, especially scrolling perf.